### PR TITLE
Admin interface for subject viewer settings

### PR DESCRIPTION
**Pull request by srallen**
Staging branch URL: https://pr-5830.pfe-preview.zooniverse.org

I added a select dropdown to the admin interface for projects to set the subject viewer to use per workflow. I also reorganized the layout because it was really inconsistent and didn't clearly label what different settings were for.

Here we see the staging Anti-Slavery Test project with its default workflow setup for the new classifier using the `multiFrame` viewer. Most workflows won't have this set yet because they haven't migrated over to the new classifier.
Screen Shot 2020-10-23 at 11 40 53 AM

Specs are updated.

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

**Comment by srallen** [comment]
ASM test can be found here: https://local.zooniverse.org:3735/admin/project_status/wgranger-test/anti-slavery-testing

File: app/pages/admin/project-status.jsx [hunk 1]
```diff
@@ -241,20 +265,44 @@ class ProjectStatus extends Component {
...
+                <label>
+                  Subject Viewer:{' '}
+                  <select
+                    id="subject-viewers"
+                    onChange={(event) => this.onChangeSubjectViewer(workflow, event)}
+                    value={workflow.configuration.subject_viewer}
+                  >
+                    <option value="none">None Selected</option>
+                    <option value="dataImage">Data Image</option>
+                    <option value="lightCurve">(D3/TESS) Light Curve</option>
```
Comment (eatyourgreens) [inline app/pages/admin/project-status.jsx:294]: This is `lightcurve` in the monorepo code but maybe we should change it there?
https://github.com/zooniverse/front-end-monorepo/blob/80ce4f31ce3ab3c1f5efd24c3f71e0d70f755198/packages/lib-classifier/src/store/Workflow/WorkflowConfiguration/WorkflowConfiguration.js#L8-L15

**Review by eatyourgreens — APPROVED** [review]
Looks good to me. We should decide whether the TESS light curve viewer should be `lightCurve` or `lightcurve`.

**Comment by eatyourgreens** [comment]
One accessibility enhancement would be to link up the headings to the controls, so that they're read by NVDA or JAWS. I can't remember if VoiceOver ignores non-form elements inside forms.
https://www.w3.org/WAI/tutorials/forms/instructions/

File: app/pages/admin/project-status.jsx (hunk 1 above)
Comment (srallen) [reply app/pages/admin/project-status.jsx:294]: It's also `lightCurve`: https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/helpers/getViewer/getViewer.js

This is the file I used as reference, so presumably it's working as `lightCurve` or else the viewer component itself wouldn't render? I'm not sure.

Comment (eatyourgreens) [reply app/pages/admin/project-status.jsx:294]: Weird. Maybe the workflow config model is wrong?

Comment (eatyourgreens) [reply app/pages/admin/project-status.jsx:294]: I did a bit of a digging and `workflow.configuration.viewerType` is used to set `subject.viewer`. I'm not sure what the difference is between that and the `getViewer` helper function.
https://github.com/zooniverse/front-end-monorepo/blob/80ce4f31ce3ab3c1f5efd24c3f71e0d70f755198/packages/lib-classifier/src/store/Subject/Subject.js#L81-L109

Comment (eatyourgreens) [reply app/pages/admin/project-status.jsx:294]: Oh, this is very convoluted. The workflow config uses the enumeration from `@helpers/subjectViewers`, which translates `lightcurve` to `lightCurve`.
https://github.com/zooniverse/front-end-monorepo/blob/80ce4f31ce3ab3c1f5efd24c3f71e0d70f755198/packages/lib-classifier/src/helpers/subjectViewers/subjectViewers.js#L8-L11

So, if I'm reading the code correctly, the workflow config stored in Panoptes is `lightcurve`, which the classifier converts internally to `lightCurve`, which is then used to lookup the viewer component in `getViewer`.

Comment (srallen) [reply app/pages/admin/project-status.jsx:294]: Thanks for looking. That is really convoluted. I think I'll update this here to use `lightcurve`, then also update `lightCurve` to be `lightcurve` where applicable in the code there so it's all consistent. The inconsistency makes for easy mistakes... like I almost did here.

Comment (srallen) [reply app/pages/admin/project-status.jsx:294]:
```suggestion
                    <option value="lightcurve">(D3/TESS) Light Curve</option>
```

Comment (eatyourgreens) [reply app/pages/admin/project-status.jsx]: Sounds like a good plan. I haven't tried this in the code, but now that `workflow.configuration.subject_viewer` is an enumeration in the model, I think the `@helpers/subjectViewers` enumeration is redundant and could be removed, along with the `workflow.configuration.viewerType` view.

Comment (srallen) [reply app/pages/admin/project-status.jsx]: Yep it's redundant, so it could be simplified.

**Pull request merged**